src/shared-components/selectorButton/style.js

```diff
@@ -23,14 +23,14 @@ export const OuterContainer = styled.div`
 
   :focus {
     outline: none;
-    ${RadioContainer} {
+    ${SelectorContainer} {
```

Comment:
you'll need to change the border-radius here based on the selector type.

![screen shot 2019-01-10 at 4 56 03 pm](https://user-images.githubusercontent.com/9381931/51006619-008a8100-14f9-11e9-8172-63cd99453648.png)